components/src/core/mobile-stepper/mobile-stepper.component.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { count } from '../../utils/test-utils';
+import { MobileStepperModule } from './mobile-stepper.module';
+import { Component } from '@angular/core';
+
+@Component({
+    template: `
+        <pxb-mobile-stepper [steps]="4" [activeIndex]="0"></pxb-mobile-stepper>
+    `,
+})
+class TestMobileStepperDot {}
+
+@Component({
+    template: `
+        <pxb-mobile-stepper [steps]="4" [activeIndex]="0" variant="progress"></pxb-mobile-stepper>
+    `,
+})
+class TestMobileStepperProgress {}
+
+describe('DotStepperComponent', () => {
+    let component: TestMobileStepperDot;
+    let fixture: ComponentFixture<TestMobileStepperDot>;
+
+    beforeEach(() => {
+        void TestBed.configureTestingModule({
+            declarations: [TestMobileStepperDot, TestMobileStepperProgress],
+            imports: [MobileStepperModule],
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(TestMobileStepperDot);
+        component = fixture.componentInstance;
+    });
+
+    it('should initialize', () => {
+        fixture.detectChanges();
+        void expect(component).toBeTruthy();
+    });
+
+    it('should render the correct number of steps', () => {
+        fixture.detectChanges();
+        const dots = fixture.nativeElement.querySelectorAll('.pxb-mobile-stepper-dot');
+        void expect(dots.length).toBe(4);
+    });
+
```

Comment:
can we test the correct active styles are applied to the activeStep?

Reply:
The class-name convention story tests that the `pxb-mobile-stepper-dot-active` class is applied.  Since the colors are provided from an external theme we won't be able to test for specific color, just class presense.

Reply:
could we check that the class is applied on the 0th item?